packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -40,20 +43,27 @@ const compressPubKey = (pubKey: string): string => {
  * A base class that can be shared between Ethers and Cosmos signers.
  */
 export class PKPBase<T = PKPBaseDefaultParams> {
-  pkpWalletProp: PKPBaseProp;
+  pkpPubKey: string;
```

Comment:
the term `pkpPubKey` essentially refers to the `uncompressedPubKey`; i don't mind them both coexists. The rationale for this is that pkpPubKey is more Ethereum-centric, while uncompressed/compressed public keys are more neutral and can be applied to different blockchains